options/locale/locale_en-US.ini

```
@@ -732,8 +732,12 @@ migrate.lfs_mirror_unsupported = Mirroring LFS objects is not supported - use 'g
 migrate.migrate_items_options = Authentication is needed to migrate items from a service that supports them.
```

```suggestion
migrate.migrate_items_options = Access Token is required to migrate additional items
```
Since we only show these options for a service that supports them now